versions/3.0.md

```diff
@@ -2323,7 +2327,9 @@ While composition offers model extensibility, it does not imply a hierarchy betw
 To support polymorphism, OpenAPI Specification adds the support of the `discriminator` field.
 When used, the `discriminator` will be the name of the property used to decide which schema definition is used to validate the structure of the model.
 As such, the `discriminator` field MUST be a required field.
-The value of the chosen property has to be the friendly name given to the model under the `definitions` property.
+There are are two ways to define the value of a discriminator for an inheriting instance.
+- Use the definition's name.
+- Override the definition's name by overriding the property with a new value. If exists, this takes precedence over the definition's name.
```

Comment:
I'm not sure what this means. Override which property - the discriminator field (`petType` in this example)? If so, how is it overridden and specified? Would I do:

```YAML
  Cat:  ## "Cat" will be used as the discriminator value
    description: A representation of a cat
    allOf:
    - $ref: '#/definitions/Pet'
    - type: object
      properties:
         petType:
              type: string
              enum:
              - Cat
          huntingSkill:
             ...
```

An example would help.